.gitignore

```diff
@@ -10,3 +10,5 @@ external/
 icons/svg-min/
 .sass-cache/
 dist/
+.DS_Store
```

Comment:
Debatable. These files are operating system specific. shouldn't be part of the repo's gitignore. Atleast thats what I think. @arthurvr  opinion?